start.py

```diff
@@ -956,10 +957,17 @@ def sigterm_handler(_signo, _stack_frame):
 
     signal.signal(signal.SIGTERM, sigterm_handler)
 
-    service_backups()
-    set_up_nginx_files(m2ee)
-    complete_start_procedure_safe_to_use_for_restart(m2ee)
-    set_up_instadeploy_if_deploy_password_is_set(m2ee)
-    start_metrics(m2ee)
-    start_nginx()
-    loop_until_process_dies(m2ee)
+    try:
+        service_backups()
+        set_up_nginx_files(m2ee)
+        complete_start_procedure_safe_to_use_for_restart(m2ee)
+        set_up_instadeploy_if_deploy_password_is_set(m2ee)
+        start_metrics(m2ee)
+        start_nginx()
+        loop_until_process_dies(m2ee)
+    except Exception:
+        x = traceback.format_exc()
+        logger.error('Starting app container failed: %s' % x)
+        callback_url = os.environ.get('BUILD_STATUS_CALLBACK_URL')
+        if callback_url:
+            requests.put(callback_url, x)
```

Comment:
Fair point on `raise`. I did not know that CF actually would receive it and
have an interpretation on it.

On second point, no. It was just a gut-feeling based on what I've learnt so
far. The intention is just to catch anything that can go wrong and record
the failure and proceed to crash.

Will update the PR.